docs/data/material/pages.ts

```diff
@@ -15,6 +15,10 @@ const pages: MuiPage[] = [
       { pathname: '/material-ui/getting-started/learn' },
       { pathname: '/material-ui/getting-started/design-resources' },
       { pathname: '/material-ui/getting-started/faq', title: 'FAQs' },
+      {
+        pathname: '/material-ui/getting-started/understand-mui-packages',
+        title: 'Understanding MUI packages',
+      },
```

Comment:
This pages doesn't exist 

https://deploy-preview-40403--material-ui.netlify.app/material-ui/getting-started/faq/

Reply:
Tackled this in https://github.com/mui/material-ui/pull/40988 👍